dev-cpp/ut2-glaze/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="proxy">
+		<email>lg3dx6fd@gmail.com</email>
+		<name>Arniiiii</name>
+		<description>Just a guy that likes C++ and uses CMake.</description>
```

Comment:
consider changing it to your info

Reply:
Would be happy to, but I'm not a maintainer, just a random bod

Reply:
ok... If I'm going to get emails about these two packages, I'm ok to update them sometimes.

Reply:
> Would be happy to, but I'm not a maintainer, just a random bod

@Rcomian If you don't intend to maintain the packages in the long run (only to do some passing-by commits), then dropping yourself from metadata.xml would be fine, the good thing is that Julien is willing to take care